config.toml

```diff
@@ -83,10 +83,11 @@ githubWebsiteRepo = "github.com/kubernetes/website"
 githubWebsiteRaw = "raw.githubusercontent.com/kubernetes/website"
 
 # param for displaying an announcement block on every page; see PR #16210
-announcement = false
+announcement = true
```

Comment:
⚠️ Commit this before merging ⚠️ 

```suggestion
announcement = false
```

Reply:
/hold
Holding to ensure we don't merge this too soon :)

Reply:
@cblecker theoretically we could merge this PR (with the flag `false` to disable the announcements) and then open a second PR when ready with wording!

Or we can do it all in this PR :D

Reply:
@celestehorgan That's what I mean. I know you have this true so that the preview shows it, but I don't want it to merge before the false flag is set.